### PR TITLE
Added a way to track if files backups and database backup succeeded

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description
Added a way to track if files backups and database backup succeeded

# Impacts
database-backup-and-maintenance script

# Delete branch after merge
YES

### CODE REVIEW

**Comment by dwlfrth** [comment]
Review me

**Comment by fdurand** [comment]
Looks good to me

File: addons/database-backup-and-maintenance.sh [hunk 1]
```diff
@@ -55,13 +55,15 @@ if ((  $BACKUPS_AVAILABLE_SPACE > (( $PF_USED_SPACE / 2 )) )); then
     if [ ! -f $BACKUP_DIRECTORY$BACKUP_PF_FILENAME ]; then
         tar -czf $current_tgz $PF_DIRECTORY --exclude=$PF_DIRECTORY'logs/*' --exclude=$PF_DIRECTORY'var/*' --exclude=$PF_DIRECTORY'.git/*'
         echo -e $BACKUP_PF_FILENAME "have been created in  $BACKUP_DIRECTORY \n"
+        echo "OK" > /usr/local/pf/var/backup_files.status
```
Comment (julsemaan) [inline addons/database-backup-and-maintenance.sh]: Shouldn't we check if the result of the tar above is good?

Comment (dwlfrth) [reply addons/database-backup-and-maintenance.sh]: How would you test that ?

Comment (julsemaan) [reply addons/database-backup-and-maintenance.sh]: check if $? is equal to 0 right after the tar

**Comment by dwlfrth** [comment]
Review me

**Pull request merged**